sal-service/src/main/java/org/ow2/proactive/sal/service/service/ByonService.java

```diff
@@ -148,25 +148,26 @@ public List<ByonNode> getByonNodes(String sessionId, String jobId) throws NotCon
      */
     public Boolean addByonNodes(String sessionId, Map<String, String> byonIdPerComponent, String jobId)
             throws NotConnectedException {
-        LOGGER.info("addByonNodes endpoint is called for job " + jobId);
+        LOGGER.info("addByonNodes endpoint is called for job {}.", jobId);
         if (!paGatewayService.isConnectionActive(sessionId)) {
             throw new NotConnectedException();
         }
         Validate.notNull(byonIdPerComponent,
                          "The received byonIdPerComponent structure is empty. Nothing to be added.");
-
-        byonIdPerComponent.forEach((byonNodeId, componentName) -> {
+        byonIdPerComponent.forEach((byonNodeId, nodeNameAndcomponentName) -> {
             ByonNode byonNode = repositoryService.getByonNode(byonNodeId);
+            String nodeName = nodeNameAndcomponentName.split("/")[0];
```

Comment:
And if your parameter does not contain "/"? You should throw a `IllegalArgumentException`